### ISSUE TITLE
[FEATURE] Randomise Users on tags search

### ISSUE THREAD

**Issue by Aadarsh805** [EddieHubCommunity/LinkFree#3380]
### Description

The Profiles on the search display are sorted alphabetically based on the profile's GitHub username
This causes all the profiles starting with the letter `A` to come on top of others
It'd be much better if this section was randomized so that more profiles can be discovered

### Screenshots

`Prakhar Goyal`'s profile is on top here as it's being sorted using the GitHub username as `Prakhar Goyal`'s username is `0xPrakharG`, it comes on top

![image](https://user-images.githubusercontent.com/95094057/212374884-d0331701-5912-423c-8c4e-ee463bea1f51.png)

**Comment by HarshDeep61034** [comment]
Great issue certainly will help in discovering new people.

**Comment by Olohitai** [comment]
Can I be assigned to this

**Comment by nanthakumaran-s** [comment]
What about sorting it based on the number of profile view?

**Comment by SaraJaoude** [comment]
> What about sorting it based on the number of profile view?

I don't think this is a good idea.  It promotes gaming the system and doesn't allow for users to discover less known profiles within this tag.  Profile views are also already done in Trending and Popular which gives this enough exposure.

Let's proceed with randomising.

**Comment by swastik959** [comment]
@SaraJaoude hi I am new to open source and need help to work on this issue

**Comment by Joelute** [comment]
Hello @SaraJaoude! I've created a PR for this issue and would love you to have a look at it and leave a review.

**Comment by SaraJaoude** [comment]
@Joelute I have now assigned this to you as it was not assigned to anyone else. 

In future, please request that the maintainers assign an Issue to you, before starting work and raising a Pull Request.  This will reduce confusion and extra work for the community and maintainers. 

Thank you for the PR - it will be reviewed shortly.

**Comment by saaagarsingh** [comment]
Hey @SaraJaoude is this issue resolved???

**Comment by SaraJaoude** [comment]
@saaagarsingh as you can see from the conversation above this Issue has been assigned to someone else who has created a PR. This PR will be looked at when maintainers can.
